planb/update_cluster.py

```diff
@@ -330,6 +330,12 @@ def step_forward(ec2: object, volume_id: str, options: dict):
     return True
 
 
+def ssh_command_works(odd_host: str):
```

Comment:
Good point.  Should we rename it as well?  I don't have a good idea right now, but maybe `test_ssh_connection`, to start with a verb?